modules/pubsub/src/main/resources/reference.conf

```diff
@@ -2,15 +2,13 @@ snowplow.defaults: {
   sources: {
     pubsub: {
       parallelPullFactor: 0.5
-      bufferMaxBytes: 10000000
-      maxAckExtensionPeriod: "1 hour"
-      minDurationPerAckExtension: "60 seconds"
-      maxDurationPerAckExtension: "600 seconds"
+      durationPerAckExtension: "60 seconds"
+      minRemainingAckDeadline: 0.1
+      maxMessagesPerPull: 1000
+      debounceRequests: "100 millis"
```

Comment:
If we don't use subscriber impl from SDK, does it also mean we can change something in our build dependencies? Perhaps right now we're still pulling some pubsub dependency we don't really need anymore.